src/store/reducers/moviesReducer.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-case-declarations */
+import {
+    LOAD_MOVIES, LOAD_MOVIES_BY_GENRE, SUBMIT_MOVIE,
+    REMOVE_MOVIE, CHANGE_FILTER } from "../actions/actionTypes";
+ import { v4 as uuidv4 } from 'uuid';
+ // ASSETS
+import { filterArray } from "../../assets/functions/util";
+
+const initialState = {
+    movies : [],
+    filter : 'RELEASE DATE',
+}
+
+const moviesReducer = (state = initialState, action) => {
+    const {movies} = state;
+    switch(action.type){
+        case LOAD_MOVIES:
+            return {...state, movies : action.payload};
+        case LOAD_MOVIES_BY_GENRE:
+            return {...state, movies : action.payload};
```

Comment:
if you have repetitive actions in reducer, you can do just:

```
         case LOAD_MOVIES:
         case LOAD_MOVIES_BY_GENRE:
             return {...state, movies : action.payload};
```

Reply:
Good to know